drs_filer/config.py

```diff
@@ -0,0 +1,35 @@
+import os
```

Comment:
Not sure about this file. I think it's really better to add this to FOCA in a generic way. It's annoying to have to have another config module to take care of after all...

Reply:
I still think this should not go in here...

Reply:
Will take care of it later, when it is added to `foca`. Deleting the file for now.